<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>799</width>
       <height>579</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>主菜单</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionFlight_enquiry"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>我的</string>
    </property>
    <addaction name="actionMy_ticket"/>
    <addaction name="actionMy_voyage_records"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFlight_enquiry">
   <property name="text">
    <string>Flight enquiry</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Book a flight ticket</string>
   </property>
  </action>
  <action name="actionRefund_a_ticket">
   <property name="text">
    <string>Refund a ticket</string>
   </property>
  </action>
  <action name="actionMy_ticket">
   <property name="text">
    <string>My ticket</string>
   </property>
  </action>
  <action name="actionMy_voyage_records">
   <property name="text">
    <string>My voyage records</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
